controller/test/units/units_test.cpp

```diff
@@ -51,6 +51,20 @@ TEST(Units, Pressure) {
 
   checkRelationalOperators(kPa);
   checkRelationalOperators(cmH2O);
+
+  Pressure pressure = kPa(2);
+  pressure += kPa(1);
+  EXPECT_FLOAT_EQ(pressure.kPa(), 3.0f);
+  pressure -= kPa(5);
+  EXPECT_FLOAT_EQ(pressure.kPa(), -2.0f);
+  pressure /= -2.0f;
+  EXPECT_FLOAT_EQ(pressure.kPa(), 1.0f);
+  pressure *= -5.0f;
+  EXPECT_FLOAT_EQ(pressure.kPa(), -5.0f);
+
+  EXPECT_FLOAT_EQ((cmH2O(1) * 10.1972f).kPa(), 1);
+  EXPECT_FLOAT_EQ((10.1972f * cmH2O(1)).kPa(), 1);
+  EXPECT_FLOAT_EQ((kPa(1) / 10.1972f).cmH2O(), 1);
```

Comment:
I am super-paranoid about having a bug in these operators, it would be pretty miserable.  Could we have some tests which check that the resulting value is not 1?  Same below.

Reply:
Makes sense, I'll build some less trivial test cases